src/services/sync/strategies/WebUntis/WebUntisSchoolyearSyncer.js

```diff
@@ -0,0 +1,488 @@
+const assert = require('assert');
+
+const WebUntisBaseSyncer = require('./WebUntisBaseSyncer');
+
+
+/**
+ * Implements syncing from WebUntis API based on the Syncer interface
+ * @class WebUntisSchoolyearSyncer
+ * @implements {Syncer}
+ */
+class WebUntisSchoolyearSyncer extends WebUntisBaseSyncer {
+	/**
+    * Constructor
+    *
+    * Disabled because of ESlint
+    *
+    * @param {*} app
+    * @param {*} stats
+    * @param {*} logger
+	* @param {*} data
+	* 
+	* Current data structure: {
+	*	webuntisConfig: {
+	*	  username: Username for the user to access WebUntis,
+	*	  password: Password for the user to access WebUntis,
+	*	  url: URL of the WebUntis endpoint,
+	*     schoolname: Identifier of the school in WebUntis
+	*   }
+	*	datatype: 'inclusive' or 'exclusive', depending on the intended semantics for courseMetadataIds
+	*	courseMetadataIds: list of metadata IDs to consider for inclusion/rejection
+	*	dryrun: collect metadata instead of synching
+	* }
+    */
+	/* constructor(app, stats, logger, data) {
+		super(app, stats, logger, data);
+	} */
+
+	/**
+	 * @see {Syncer#respondsTo}
+	 */
+	static respondsTo(target) {
+		return target === 'webuntis';
+	}
+
+	static params(params, data = {}) {
+		if (!params || !data) {
+			return false;
+		}
+
+		const query = (params || {}).query || {}; // object containing url, username, password, and schoolname
+
+		const validQuery = (
+			query.username != '' &&
+			query.url != '' &&
+			query.password != '' &&
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4061937)